backend/fms_core/models/sample_lineage.py

```diff
@@ -27,23 +25,27 @@ def add_error(field: str, error: str):
 
         protocol_name = self.process_measurement.process.protocol.name
         if protocol_name == 'Extraction':
-            Sample = apps.get_model("fms_core", "Sample")
-            if self.child.sample_kind.name not in Sample.BIOSPECIMEN_TYPES_NA:
+            DerivedSample = apps.get_model("fms_core", "DerivedSample")
```

Comment:
same comment, try to import the models with import and not with the apps.get_model

Reply:
ok